Bugsnag/Assets/Bugsnag/Editor/BugsnagAddScriptingSymbol.cs

```diff
@@ -1,38 +1,44 @@
 using UnityEditor;
+#if UNITY_6000_0_OR_NEWER
+using UnityEditor.Build;
+#endif
 using UnityEngine;
```

Comment:
Are all these includes (and the if-def) still required now we've abstracted to the compat class for some of it?

Reply:
ah yes, my mistake, well spotted